.travis.yml

```diff
@@ -19,29 +19,35 @@ before_install:
   - gem --version
 
 script:
-  - export QFIELD_SDK_VERSION=20190123
+  - export QFIELD_SDK_VERSION=20190128
   - echo "travis_fold:start:docker-pull"
   - docker pull opengisch/qfield-sdk:${QFIELD_SDK_VERSION}
   - echo "travis_fold:end:docker-pull"
   - docker run -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e "ARCH=${ARCH}" -e "STOREPASS=${STOREPASS}" -e "KEYNAME=${KEYNAME}" -e "KEYPASS=${KEYPASS}" -e "VERSION=${TRAVIS_TAG}" opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
   - ./scripts/upload-artifacts.sh
 
 jobs:
+  allow_failures:
+    - name: testing
```

Comment:
what's this for?

Reply:
test is currently failing because of a missing declare metatype in QGIS core (PR merged, waiting for new image to be built)